shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1411,7 +1411,7 @@ void WebContents::RenderProcessGone(base::TerminationStatus status) {
   Emit("crashed", status == base::TERMINATION_STATUS_PROCESS_WAS_KILLED);
 
   // User might destroy WebContents in the crashed event.
-  if (!weak_this)
+  if (!weak_this || !web_contents())
```

Comment:
Can this ever be called when `web_contents()` is nullptr? RenderProcessGone is a WebContentsObserver method, so if we've called `Observe(nullptr)` we shouldn't ever get an observer method call. If `Emit("crashed")` destroyed the WebContents, then the weak_this check will be enough to guard this.